swift/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ghcr.io/dependabot/dependabot-updater-core
+ARG TARGETARCH
+
+ENV PATH="${PATH}:/opt/swift/usr/bin"
+
+# OS dependencies
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+    binutils \
+    libc6-dev \
+    libcurl4 \
+    libedit2 \
+    libgcc-9-dev \
+    libpython2.7 \
+    libsqlite3-0 \
+    libstdc++-9-dev \
+    libxml2 \
+    libz3-dev \
+    pkg-config \
+    tzdata \
+    uuid-dev \
```

Comment:
There's some dependencies I'm curious about, like sqlite3 and libpython2.7. Are these all needed?

Reply:
I got no idea. I copied this from the table at https://www.swift.org/download/#using-downloads (Linux section).

Reply:
`libpython2.7` looks suspicious, as Swift already moves to Python 3. Also, Python is only required for LLDB, so as long as we don’t use the debugger it should be unnecessary.

`sqlite3` is required for SwiftPM.

Reply:
What version of Ubuntu is this targeting? 22.04 uses Python 3

Reply:
🤦 I see below it's using 20.04 - why not use Jammy here?

Reply:
Unfortunately our base image still uses 20.04 so we cannot move on just yet. See https://github.com/dependabot/dependabot-core/pull/5030.